app/domain/operations/data_fixes/create_v3_individual_market_evidences.rb

```diff
@@ -0,0 +1,125 @@
+# frozen_string_literal: true
+
+require 'dry/monads'
+require 'dry/monads/do'
+
+# ::Operations::DataFixes::CreateIndividualMarketEvidences.new.call({application_hbx_id: application_hbx_id})
+module Operations
+  module DataFixes
+    # This operation creates Individual Market evidences.
+    class CreateV3IndividualMarketEvidences
```

Comment:
As the part of QHP migration, I am migrating all the verification types to the application. We need to generate a report to fetch exactly what is missing as the part of the IndividualMarketEvidences. Can you move this script out of this PR, based on the report we will create a new PR for IndividualMarketEvidences.

Reply:
Can you also generate a report to list out the missing IndividualMarketEvidences?